src/main/scala/com/mwz/sonar/scala/scapegoat/ScapegoatRulesRepository.scala

```diff
@@ -0,0 +1,50 @@
+package com.mwz.sonar.scala
+package scapegoat
+
+import inspections.{Level, ScapegoatInspection}
+
+import org.sonar.api.server.rule.RulesDefinition
+import org.sonar.api.rule.RuleStatus
+import org.sonar.api.rule.Severity
+import org.sonar.api.rules.RuleType
+
+/** Defines a rules repository for the Scapegoat inspections */
+class ScapegoatRulesRepository extends RulesDefinition {
+
+  /** Defines the rules in the repository */
+  override def define(context: RulesDefinition.Context): Unit = {
+    // crete an empty repository
+    val repository =
+      context
+        .createRepository(ScapegoatRulesRepository.RepositoryKey, Scala.LanguageKey)
+        .setName(ScapegoatRulesRepository.RepositoryName)
+
+    // register each scapegoat inspection as a repository rule
+    for (inspection <- ScapegoatInspection.AllScapegoatInspections) {
```

Comment:
I'd prefer if we did that `ScapegoatInspection.AllScapegoatInspections foreach { inspection =>`, otherwise Scalastyle will complain about the use of parentheses in a for comprehension, see https://sonar.sonar-scala.com/coding_rules#rule_key=Scalastyle%3Ascalastyle_ForBraceChecker.

Reply:
Ok